src/makeConfig.ts

```diff
@@ -0,0 +1,69 @@
+import { Options, FetchContextTypes, OptionsMaybeURL, NoUrlOptions } from './types'
+import { isString, isObject, pullOutRequestInit } from "./utils"
+
+export const DefaultOptions: NoUrlOptions = {
+  onMount: false
+}
+
+function makeConfig(context: FetchContextTypes, urlOrOptions?: string | OptionsMaybeURL, optionsNoURLs?: NoUrlOptions): Options {
+  let options: Partial<Options> = {}
+  let requestInit: RequestInit = {}
+
+  if (isObject(urlOrOptions) && isObject(optionsNoURLs)) {
+    // TODO: I could not get the utils/invariant to work with the condition isObject(urlOrOptions) && isObject(optionsNoURLs)
+    throw new Error('You cannot have a 2nd parameter of useFetch when your first argument is a object config.')
```

Comment:
You can, you just have to make it a boolean. Also remember, it errors on `false`

Reply:
So it would be `invariant(isObject(urlOrOptions) || isObject(optionsNoURLs), 'You cannot have a 2nd parameter of useFetch when your first argument is a object config.')`

Reply:
haha, yes, I got confused and thought it errored on truthy

Reply:
your invariant expression won't work because the urlOrOption can be an object.  I do find this being falsy confusing

Reply:
ok, that worked

`invariant(!(isObject(urlOrOptions) && isObject(optionsNoURLs)), 'You cannot have a 2nd parameter of useFetch when your first argument is an object config.')`

Reply:
I know, it's just how facebook does it and it's nice being a one-liner

Reply:
whoops, what you said is what I meant.

Reply:
> ok, that worked
> 
> `invariant(!(isObject(urlOrOptions) && isObject(optionsNoURLs)), 'You cannot have a 2nd parameter of useFetch when your first argument is an object config.')`

whoops, this is what I meant